iree/compiler/Conversion/LinalgToSPIRV/Passes.cpp

```diff
@@ -50,22 +51,15 @@ namespace mlir {
 namespace iree_compiler {
 
 namespace {
-/// Command line options for use with SPIR-V code-generation pass pipeline.
-struct SPIRVCodegenClOpts : public PassPipelineOptions<SPIRVCodegenClOpts> {
-  ListOption<int64_t> workgroupSize{
-      *this, "workgroup-size",
-      llvm::cl::desc(
-          "Number of workgroups to dispatch for the SPIR-V module; at most "
-          "three integers standarding for the x, y, and z dimension; "
-          "additional arguments will be ignored (used only for testing)"),
-      llvm::cl::ZeroOrMore, llvm::cl::MiscFlags::CommaSeparated};
-  ListOption<int64_t> tileSizes{
-      *this, "tile-sizes", llvm::cl::desc("Set tile sizes to use"),
-      llvm::cl::ZeroOrMore, llvm::cl::MiscFlags::CommaSeparated};
-  Option<bool> useWorkgroupMemory{
-      *this, "use-workgroup-memory",
+
+/// Linalg to SPIR-V pass pipeline options. In theory this is a superset of all
+/// options in all passes in the pipeline. Adding those based on need, and they
+/// should be needed only for testing.
```

Comment:
? I thought these ones are for real and the command line ones are for testing purpose?

Reply:
This is only for lit-testing. The real options are the `SPIRVCodegenOptions` . Maybe the placement is misleading.